src/options.rs

```diff
@@ -901,9 +901,9 @@ impl Options {
 
                 if e.is::<ClientNoCredentialsError>() {
                     let project_dir = get_project_path(None, true).await?;
-                    let message = if project_dir.is_some() {
+                    let message = if let Some(project_dir) = project_dir {
                         format!(
-                            "project is not initialized and no connection options \
+                            "project is not initialized at {project_dir:?} and no connection options \
```

Comment:
Perhaps we need to clarify that it's not initialized in the specific directory or in any of the parent directories

Reply:
Agreed -- although I think I should close this PR in favour of the more verbose project init message discussed on Slack.